style.css

```diff
@@ -9,10 +9,38 @@
  * minimal changes to the HTML itself.
  */
 
+ header {
+ 	background-color: #000066;
+ 	color: white;
+ 	font-family: Courier New, Courier, monospace;
```

Comment:
> To avoid mistakes in escaping, it is recommended to quote font family names that contain white space, digits, or punctuation characters other than hyphens:

https://www.w3.org/TR/CSS2/fonts.html#font-family-prop

Courier New works in this case but it is recommended to quote it (`"Courier New "`) to avoid potential issues because it has a space in the name.